### ISSUE TITLE
Allow us to close portions of the debugger

### ISSUE THREAD

**Issue by Jarrio** [jhen0409/react-native-debugger#86]
- [x] I'm using the latest version of react-native-debugger
- [x] I have read [README.md](https://github.com/jhen0409/react-native-debugger/blob/master/README.md)

** Please provide the following information if you found some bugs or have question:

React Native version: [0.40.0]
Platform: [Android]
Is real device of platform: [No]? (Phrasing of this is odd - I'm using an emulator atm)
Operating System: [Windows]

There are times when I'm doing UI work and there are times when all I want is a console. Please allow us the option to close portions of the debugger. It's not very space efficient to hit the `Toggle React-Tools` and it turns off but the frame is still there and cannot be shrunk to a smaller size.

![image](https://cloud.githubusercontent.com/assets/748557/26642165/a4215a78-4624-11e7-8e3d-095515df6897.png)

As you can see from the above screenshot it kind of takes up unnecessary space when tiling windows in specific ways

**Comment by jhen0409** [comment]
Yeah, I think it would be great to have a menu item for undock the devtools and hide the debugger window.
